core/app_administration.go

```diff
@@ -560,6 +560,68 @@ func (app *application) admGetAccount(accountID string) (*model.Account, error)
 	return app.getAccount(accountID)
 }
 
+func (app *application) admGetAccountSystemConfigs(appID string, orgID string, accountID string, l *logs.Log) (map[string]interface{}, error) {
+	//find the account
+	account, err := app.storage.FindAccountByID(nil, accountID)
+	if err != nil {
+		return nil, errors.WrapErrorAction(logutils.ActionFind, model.TypeAccountSystemConfigs, &logutils.FieldArgs{"account_id": accountID}, err)
+	}
+	if account == nil {
+		return nil, errors.WrapErrorData(logutils.StatusMissing, model.TypeAccountSystemConfigs, &logutils.FieldArgs{"account_id": accountID}, err)
+	}
+	if account.AppOrg.Application.ID != appID || account.AppOrg.Organization.ID != orgID {
+		l.Warnf("someone is trying to get system configs for %s for different app/org", accountID)
```

Comment:
This is a very good point, but I think it actually applies to all admin endpoints. I would like to ensure that the admin ID is added to the log context so that it is always printed in the request completed log and can be re-associated through the trace ID. I have opened issue #562 to address this.

Reply:
Sounds good, thanks!